### PR TITLE
PLAT-516: parallel call of two ordered methods

### PR DESCRIPTION
**Pull request by IgorMarnat**
Added test TestVirtual_Method_WithoutExecutor_Ordered which checks that ordered methods are executed sequentially, second is blocked until the first one is done.

### CODE REVIEW

File: ledger-core/virtual/integration/method_test.go [hunk 1]
```diff
@@ -343,6 +344,90 @@ func TestVirtual_Method_WithoutExecutor_Unordered(t *testing.T) {
...
+			server.SendPayload(ctx, &pl)
+		}
+	}
+	awaitFullStop := server.Journal.WaitStopOf(&execute.SMExecute{}, 2)
+	testutils.WaitSignalsTimed(t, 10*time.Second, awaitFullStop)
+	testutils.WaitSignalsTimed(t, 10*time.Second, server.Journal.WaitAllAsyncCallsDone())
+	{
+		assert.Equal(t, 2, typedChecker.VCallResult.Count())
+
+		if !server.PublisherMock.WaitCount(countBefore+2, 10*time.Second) {
```
Comment (bigbes) [inline ledger-core/virtual/integration/method_test.go:423]: that part is not needed

File: ledger-core/virtual/integration/method_test.go [hunk 2]
```diff
@@ -343,6 +344,90 @@ func TestVirtual_Method_WithoutExecutor_Unordered(t *testing.T) {
...
+			})
+			runnerMock.AddExecutionClassify(key, contract.MethodIsolation{
+				Interference: interferenceFlag,
+				State:        stateFlag,
+			}, nil)
+
+			server.SendPayload(ctx, &pl)
+		}
+	}
+	awaitFullStop := server.Journal.WaitStopOf(&execute.SMExecute{}, 2)
```
Comment (bigbes) [inline ledger-core/virtual/integration/method_test.go:417]: please move that part to place before messages were sent (as we discussed before)

File: ledger-core/virtual/integration/method_test.go (hunk 1 above)
Comment (IgorMarnat) [reply ledger-core/virtual/integration/method_test.go]: Fixed

File: ledger-core/virtual/integration/method_test.go (hunk 2 above)
Comment (IgorMarnat) [reply ledger-core/virtual/integration/method_test.go]: Right. Done.

**Pull request merged**